py_swf/clients/workflow.py

```diff
@@ -348,7 +362,7 @@ def _count_closed_workflow_executions(
 
 def _build_time_filter_dict(oldest_start_date=None, latest_start_date=None, oldest_close_date=None, latest_close_date=None):
     """
-    Build time_filter_dict for count open/closed workflow executions.
+    Build time_filter_dict for calls to _count_closed_workflow_executions and _count_open_workflow_executions.
     Return result is a dict: {time_filter_type : filter_dict}
     filter_dict must contains key 'oldestDate'.
```

Comment:
Ah I did separate the count by start_date and close_date. Take count_closed_workflow_by_id_and_start_time  and count_closed_workflow_by_id_and_close_time by example. Since the function args make sure we only get start_date or close_date, that is why I remove the validation.